ambry-api/src/main/java/com/github/ambry/config/NetworkConfig.java

```diff
@@ -119,6 +122,35 @@ public class NetworkConfig {
   @Default("false")
   public final boolean selectorUseDirectBuffers;
 
+  /**
+   * The type of queue to use to hold requests waiting to be processed.
+   * TODO: Do tests to tune controlled delay queue parameters before switching the request queue type to {@link RequestQueueType#ADAPTIVE_LIFO_CO_DEL}
+   */
+  @Config("request.queue.type")
+  @Default("BASIC_FIFO")
+  public final RequestQueueType requestQueueType;
+
+  /**
+   * The maximum time in milliseconds a request is allowed to spend in the queue waiting to be processed.
+   */
+  @Config("request.queue.timeout.ms")
+  @Default("2000")
+  public final int requestQueueTimeoutMs;
+
+  /**
+   * The maximum queue delay that is allowed for a request when using Controlled delay algorithm.
+   */
+  @Config("adaptive.lifo.queue.codel.target.delay.ms")
+  @Default("100")
+  public final int adaptiveLifoQueueCodelTargetDelayMs;
+
+  /**
+   * The threshold over which we switch the request queue from FIFO to LIFO mode.
+   */
+  @Config("adaptive.lifo.queue.threshold")
+  @Default("0.7")
+  public final double adaptiveLifoQueueThreshold;
```

Comment:
Is those `@Default `actually in-use/needed? Since i see we explicitly set the defaultValue in constructor like below:
` adaptiveLifoQueueThreshold = verifiableProperties.getDoubleInRange("adaptive.queue.lifo.threshold", 0.7, 0.0, 1.0);`

Reply:
Will check once. I copied from other configs but like you said`@Default` annotation may not be in use.